### PR TITLE
Sejin EAT-1010R2 keyboard

### PR DESCRIPTION
**Pull request by DmNosachev**
## Description
Sejin EAT-1010R2 keyboard, STM32F103 Blue Pill handwired to the membrane.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/sejin_eat1010r2/config.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* key matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 16
+
```
Comment (zvecr) [inline keyboards/handwired/sejin_eat1010r2/config.h:25]:
```suggestion
```

File: keyboards/handwired/sejin_eat1010r2/halconf.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* Copyright 2020 QMK
```
Comment (zvecr) [inline keyboards/handwired/sejin_eat1010r2/halconf.h:1]: File should be deleted

File: keyboards/handwired/sejin_eat1010r2/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+/*
+void matrix_init_user(void) {
+}
+void matrix_scan_user(void) {
+}
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (zvecr) [inline keyboards/handwired/sejin_eat1010r2/keymaps/default/keymap.c:58]:
```suggestion
```

File: keyboards/handwired/sejin_eat1010r2/mcuconf.h [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* Copyright 2020 QMK
```
Comment (zvecr) [inline keyboards/handwired/sejin_eat1010r2/mcuconf.h:1]: File should be deleted

File: keyboards/handwired/sejin_eat1010r2/sejin_eat1010r2.c [hunk 5]
```diff
@@ -0,0 +1,39 @@
+/* Copyright 2023 DmNosachev
```
Comment (zvecr) [inline keyboards/handwired/sejin_eat1010r2/sejin_eat1010r2.c:1]: File should be deleted

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by waffle87** [comment]
Unsure why you closed #19802, but the changes I suggested there still need to be applied to this one.

**Comment by drashna** [comment]
> Unsure why you closed #19802, but the changes I suggested there still need to be applied to this one.

To pile on that, PRs shouldn't be closed and reopened.  If there is an issue with a PR, there is a good chance that it is something that we've run into as well, and may be able to help fix.

**Comment by DmNosachev** [comment]
Sorry for the inconvenience. I'm still insecure with complex Git actions. I closed the previous PR because I got confused about the commits in my fork.

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/sejin_eat1010r2/info.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        "midi": false,
+        "mousekey": true,
+        "nkro": false,
+        "rgblight": false,
+        "unicode": false,
+        "bluetooth": false,
+        "backlight": false,
+        "sleep_led": false
+    },
+	"debounce": 5,
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/info.json:38]:
```suggestion
    "diode_direction": "COL2ROW",
    "development_board": "bluepill",
    "matrix_pins": {
        "cols": ["B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "C13", "C14", "C15", "A0", "A1", "A2", "A3", "A4"],
        "rows": ["B11", "B12", "B13", "B14", "B15", "A8", "A9", "A10"],
        "ghost": true
    },
    "indicators": {
        "caps_lock": "B1",
        "num_lock": "B0",
        "scroll_lock": "B10"
    },
    "features": {
        "bootmagic": true,
        "command": false,
        "console": false,
        "extrakey": true,
        "mousekey": true,
        "nkro": false
    },
    "bootmagic" {
        "matrix": [1, 0]
    },
```

File: keyboards/handwired/sejin_eat1010r2/config.h [hunk 7]
```diff
@@ -0,0 +1,22 @@
+/*
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/config.h:1]: This file can be removed with my suggestion in `info.json`.

File: keyboards/handwired/sejin_eat1010r2/readme.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+* Keyboard Maintainer: [DmNosachev](https://github.com/DmNosachev)
+* Hardware Supported: Sejin EAT-1010R2 (membrane keyboard, Sejin
+Futaba-mount dome, Asian 101 ISO layout), Blue/Black Pill STM32F103 MCU 
+oard. Alternatevely you can use any MCU which is supported by QMK and
+has 27 or more IO pins
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb handwired/sejin_eat1010r2 -km default
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/readme.md:11]:
```suggestion
    make handwired/sejin_eat1010r2:default
    
Flashing example for this keyboard:

    make handwired/sejin_eat1010r2:default:flash

```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/handwired/sejin_eat1010r2/info.json [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false
+    },
+    "bootmagic" {
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/info.json:31]: Sorry, typo'd this.
```suggestion
    "bootmagic": {
```

**Review by waffle87 — APPROVED** [review]

**Comment by DmNosachev** [comment]
I see а message 'Changes requested' ('zvecr requested changes'), but his changes are marked as 'Outdated'. What should I do about these messages?

**Pull request merged**